TheEthicalHackerCup/Assets/Scripts/Learning/QuestionModel.cs

```diff
@@ -0,0 +1,393 @@
+using Codice.CM.Common.Serialization;
+using NSubstitute.Core;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Learning
+{
+
+    public class QuestionStateUpdatedEvent : EventArgs
+    {
```

Comment:
Consider putting question related classes into different files in a new subfolder in Scripts/Learning/Question
Not 100% needed per se, but it would help with file searchability and short files are easier to read than long ones